src/Umbraco.Web.UI.Client/src/views/common/dialogs/iconpicker.html

```diff
@@ -1,63 +1,84 @@
+                <option value="color-blue">Blue</option>
+                <option value="color-light-blue">Light Blue</option>
+                <option value="color-cyan">Cyan</option>
+                <option value="color-green">Green</option>
+                <option value="color-light-green">Light Green</option>
+                <option value="color-lime">Lime</option>
+                <!--<option value="color-yellow">Yellow</option>-->
+                <option value="color-amber">Amber</option>
+                <option value="color-orange">Orange</option>
+                <option value="color-deep-orange">Deep Orange</option>
+                <option value="color-red">Red</option>
+                <option value="color-pink">Pink</option>
+                <option value="color-purple">Purple</option>
+                <option value="color-deep-purple">Deep Purple</option>
+                <option value="color-indigo">Indigo</option>
+            </select>
+            <!--<select ng-model="color" class="input-block-level">
+                <option value="">Black</option>
+                <option value="color-green">Green</option>
+                <option value="color-yellow">Yellow</option>
+                <option value="color-orange">Orange</option>
+                <option value="color-blue">Blue</option>
+                <option value="color-red">Red</option>
+            </select>-->
+
+            <!--<select class="input-block-level">
+                <option value="">Black</option>
+                <option value="color-green">Green</option>
+                <option value="color-blue">Turquoise</option>
+                <option value="color-purple">Purple</option>
+                <option value="color-red">Red</option>
+                <option value="color-orange">Orange</option>
+                <option value="color-sky-blue">Sky blue</option>
+                <option value="color-yellow">Yellow</option>
+            </select>-->
 
         </div>
 
         <div class="umb-loader" ng-hide="icons"></div>
 
         <div class="umb-control-group">
-	        <ul class="umb-thumbnails thumbnails" ng-class="color">
-		        <li class="span1" ng-repeat="icon in icons|filter: searchTerm">
-				    <a href="#" title="{{icon}}" class="thumbnail" ng-click="submitClass(icon)" prevent-default>
-						<i class="{{icon}} large" style="font-size: 32px"></i>
-		            </a>
-		        </li>
-		    </ul>
-		</div>
-
-	</div>
-	
-	<div class="umb-panel-footer" >
-		<div class="umb-el-wrap umb-panel-buttons">
-	        <div class="btn-toolbar umb-btn-toolbar pull-right">
-	        	<a href ng-click="close()" class="btn btn-link">
-	        		<localize key="general_cancel">Cancel</localize>
-				</a>
-	        </div>
-		</div>
-	</div>
+            <ul class="umb-thumbnails thumbnails" ng-class="color">
+                <li class="icon" ng-repeat="icon in icons | filter:searchTerm track by $id(icon)">
```

Comment:
Notice track by. It seems to make a big different when having more icons than included in the core, e.g. try to install my package Material Design Icon Pack https://our.umbraco.org/projects/backoffice-extensions/material-design-icon-pack/ and you should be able to see a difference.